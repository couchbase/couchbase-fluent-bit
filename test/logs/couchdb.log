[couchdb:info,2021-03-09T17:32:01.217Z,couchdb_ns_1@cb.local:cb_couch_sup<0.143.0>:couch_log:info:41]Apache CouchDB has started on http://0.0.0.0:8092/
[couchdb:info,2021-03-09T17:33:00.994Z,couchdb_ns_1@cb.local:couch_query_logger<0.195.0>:couch_log:info:41]Query-Volume
---
[couchdb:info,2021-03-09T17:33:50.567Z,couchdb_ns_1@cb.local:<0.555.0>:couch_log:info:41]Deleting couch file "/opt/couchbase/var/lib/couchbase/data/default/master.couch.1.compact" with renaming it to "/opt/couchbase/var/lib/couchbase/data/.delete/8e547031d55d68d4e576f4f5b3375024"
[couchdb:info,2021-03-09T17:33:50.955Z,couchdb_ns_1@cb.local:couch_view<0.242.0>:couch_log:info:41]Deleting couch file "/opt/couchbase/var/lib/couchbase/data/.default/master_temp" with renaming it to "/opt/couchbase/var/lib/couchbase/data/.delete/075c3c2f87a3fc9611231cf80451d465"
[couchdb:info,2021-03-09T17:34:00.995Z,couchdb_ns_1@cb.local:couch_query_logger<0.195.0>:couch_log:info:41]Query-Volume
---
