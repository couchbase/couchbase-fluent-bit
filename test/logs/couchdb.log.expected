/fluent-bit/test/logs/couchdb.log.actual: [1615311121.217000000, {"file_path":"/fluent-bit/test/logs","file":"couchdb.log","logshipper":"couchbase.sidecar.fluentbit","timestamp":"2021-03-09T17:32:01.217Z","pod":{"namespace":"default","name":"test-pod","uid":"unknown"},"couchbase":{"server.server":"unknown","analytics":"false","data":"false","eventing":"false","index":"false","query":"false","search":"false","cluster":"test-cluster","logging.version":"99","operator.version":"unknown","server.version":"unknown","node":"cb-node.test-cluster.default.svc","node-config":"unknown"},"logger":"couchdb","fluentbit.version":"99","message":"couchdb_ns_1@cb.local:cb_couch_sup<0.143.0>:couch_log:info:41]Apache CouchDB has started on http://0.0.0.0:8092/","filename":"/fluent-bit/test/logs/couchdb.log","hostname":"test-container","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/couchdb.log.actual: [1615311180.994000000, {"file_path":"/fluent-bit/test/logs","file":"couchdb.log","logshipper":"couchbase.sidecar.fluentbit","timestamp":"2021-03-09T17:33:00.994Z","pod":{"namespace":"default","name":"test-pod","uid":"unknown"},"couchbase":{"server.server":"unknown","analytics":"false","data":"false","eventing":"false","index":"false","query":"false","search":"false","cluster":"test-cluster","logging.version":"99","operator.version":"unknown","server.version":"unknown","node":"cb-node.test-cluster.default.svc","node-config":"unknown"},"logger":"couchdb","fluentbit.version":"99","message":"couchdb_ns_1@cb.local:couch_query_logger<0.195.0>:couch_log:info:41]Query-Volume\n---","filename":"/fluent-bit/test/logs/couchdb.log","hostname":"test-container","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/couchdb.log.actual: [1615311230.567000000, {"file_path":"/fluent-bit/test/logs","file":"couchdb.log","logshipper":"couchbase.sidecar.fluentbit","timestamp":"2021-03-09T17:33:50.567Z","pod":{"namespace":"default","name":"test-pod","uid":"unknown"},"couchbase":{"server.server":"unknown","analytics":"false","data":"false","eventing":"false","index":"false","query":"false","search":"false","cluster":"test-cluster","logging.version":"99","operator.version":"unknown","server.version":"unknown","node":"cb-node.test-cluster.default.svc","node-config":"unknown"},"logger":"couchdb","fluentbit.version":"99","message":"couchdb_ns_1@cb.local:<0.555.0>:couch_log:info:41]Deleting couch file \"/opt/couchbase/var/lib/couchbase/data/default/master.couch.1.compact\" with renaming it to \"/opt/couchbase/var/lib/couchbase/data/.delete/8e547031d55d68d4e576f4f5b3375024\"","filename":"/fluent-bit/test/logs/couchdb.log","hostname":"test-container","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/couchdb.log.actual: [1615311230.955000000, {"file_path":"/fluent-bit/test/logs","file":"couchdb.log","logshipper":"couchbase.sidecar.fluentbit","timestamp":"2021-03-09T17:33:50.955Z","pod":{"namespace":"default","name":"test-pod","uid":"unknown"},"couchbase":{"server.server":"unknown","analytics":"false","data":"false","eventing":"false","index":"false","query":"false","search":"false","cluster":"test-cluster","logging.version":"99","operator.version":"unknown","server.version":"unknown","node":"cb-node.test-cluster.default.svc","node-config":"unknown"},"logger":"couchdb","fluentbit.version":"99","message":"couchdb_ns_1@cb.local:couch_view<0.242.0>:couch_log:info:41]Deleting couch file \"/opt/couchbase/var/lib/couchbase/data/.default/master_temp\" with renaming it to \"/opt/couchbase/var/lib/couchbase/data/.delete/075c3c2f87a3fc9611231cf80451d465\"","filename":"/fluent-bit/test/logs/couchdb.log","hostname":"test-container","level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/couchdb.log.actual: [1615311240.995000000, {"file_path":"/fluent-bit/test/logs","file":"couchdb.log","logshipper":"couchbase.sidecar.fluentbit","timestamp":"2021-03-09T17:34:00.995Z","pod":{"namespace":"default","name":"test-pod","uid":"unknown"},"couchbase":{"server.server":"unknown","analytics":"false","data":"false","eventing":"false","index":"false","query":"false","search":"false","cluster":"test-cluster","logging.version":"99","operator.version":"unknown","server.version":"unknown","node":"cb-node.test-cluster.default.svc","node-config":"unknown"},"logger":"couchdb","fluentbit.version":"99","message":"couchdb_ns_1@cb.local:couch_query_logger<0.195.0>:couch_log:info:41]Query-Volume\n---","filename":"/fluent-bit/test/logs/couchdb.log","hostname":"test-container","level":"INFO","level_numeric":"20"}]
